node/src/components/block_proposer.rs

```diff
@@ -86,28 +78,30 @@ impl Display for Event {
 }
 
 type DeployCollection = HashMap<DeployHash, DeployHeader>;
-pub type ProtoBlockCollection = HashMap<ProtoBlockHash, DeployCollection>;
+type FinalizationQueue = HashMap<u64, Vec<DeployHash>>;
+type RequestQueue = HashMap<u64, Vec<ListForInclusionRequest>>;
```

Comment:
Can you create a local type alias `type BlockHeight = u64`? Or even better, put it in some common module like `types.rs` because we reuse it so much.

Reply:
And also, please add a short comment on what those collections are.

Reply:
> Can you create a local type alias `type BlockHeight = u64`? Or even better, put it in some common module like `types.rs` because we reuse it so much.

Sounds like a good idea, but in order to be consistent, this should be done across the whole codebase. I think it would make sense to make this a separate PR (finding all the places where `u64` is used for block heights could be time consuming).

Reply:
Ok, we can do it in a separate PR. For now, can you add a local type alias?